### DISCUSSION TITLE
Can I start wezterm without any window?

### DISCUSSION THREAD

**Discussion by HIDE-r** [wezterm/wezterm#4113] [Q&A]
I want to use wezterm to execute a command, But I don't want to show a window widget. 

```
wezterm --config enable_wayland=false start --always-new-process bash -vc 'sleep 3 ;xdotool key super+alt+c; exit '
```
![image](https://github.com/wez/wezterm/assets/51672711/9c439f79-d8f7-48b8-9a2f-1ace6d1a41e6)

As you see, this will open a window which I don't want it show

**Comment by wez** [comment]
I'm not sure what you're trying to do here; the whole purpose of wezterm is to be a terminal emulator window.
Can you share more details about why you're not just using `bash -vc 'sleep 3 ;xdotool key super+alt+c'` directly?

**Reply by HIDE-r** [reply]
Thank for your reply~ 

Because my wayland compositor(Hyprland) can not correctly process some application global shortcut. Normally it can be fixed by using application command line flag. But some application does not have command line flag. 

But last week,  I found that the global shortcut can be triggered if the cursor focuses on xwayland window widget. So I have come up with a new workaround idea, firstly I create a xwayland window (I set `--config enable_wayland=false` for this), and then use `xdotool`  to simulate me pressing the global shortcut key (`xdotool key super+alt+c'`).

It can work ! But it must sleep some time at startup (I dont't know why, I guess it maybe need to wait for window initialization). and another question is will cause a window popup. So I think whether the terminal can hide the window.

**Reply by wez** [reply]
This request is out of scope for wezterm, sorry!

**Reply by HIDE-r** [reply]
Okay. Got it. I'm just curious if that's an option. Thanks for your help~

**Comment by HIDE-r** [comment]
>
